src/WWT.Providers/Providers/MartianTile2Provider.cs

```diff
@@ -120,34 +120,16 @@ public override void Run(IWwtContext context)
 
             }
 
-
-            string filename = String.Format(DSSTileCache + "\\wwtcache\\mars\\{3}\\{0}\\{2}\\{1}_{2}.png", level, tileX, tileY, id);
-            string path = String.Format(DSSTileCache + "\\wwtcache\\mars\\{3}\\{0}\\{2}", level, tileX, tileY, id);
-
+            string filename = $@"{_options.DssToastPng}\wwtcache\mars\{id}\{level}\{tileY}\{tileX}_{tileY}.png";
 
             if (!File.Exists(filename))
             {
-                //try
-                //{
-                //    if (!Directory.Exists(filename))
-                //    {
-                //        Directory.CreateDirectory(path);
-                //    }
-
-                //    WebClient webclient = new WebClient();
-
-                //    string url = string.Format("http://wwt.nasa.gov/wwt/p/{0}/{1}/{2}/{3}{4}", dataset, level, tileX, tileY, type);
-
-                //    webclient.DownloadFile(url, filename);
-                //}
-                //catch
-                // {
                 context.Response.StatusCode = 404;
```

Comment:
We should add a comment here mentioning that those codepath used to try to download data from `wwt.nasa.gov`, including the specific URL that used to be used. Since the file that's being checked for is a local cache file, when we rebuild the servers this endpoint will *always* 404 out unless I find a backup of the original data.